viewer/packages/360-images/src/entity/Image360VisualizationBox.ts

```diff
@@ -108,6 +122,12 @@ export class Image360VisualizationBox implements Image360Visualization {
     this._visualizationMesh.visible = this._visualizationState.visible;
     this._sceneHandler.addCustomObject(this._visualizationMesh);
 
+    if (this._annotations) {
+      this._annotations.forEach(a => {
+        this._visualizationMesh!.add(a);
```

Comment:
If you pull out this._visualizationMesh to a var in the beginning of this method, you don't have to `!` it because if is guarded by the first if in the method.